drivers/flash/Kconfig.simulator

```diff
@@ -26,6 +26,13 @@ config FLASH_SIMULATOR_DOUBLE_WRITES
 	 If selected, writing to a non-erased program unit will succeed, otherwise, it will return an error.
 	 Keep in mind that write operations can only pull bits to zero, regardless.
 
+config FLASH_SIMULATOR_STUB
```

Comment:
Maybe we could select variant of flash simulator using a choice?
That could make it easier to understand. We could also select proper source file relying on selected choice option

Reply:
The choice symbol could also be used for Kconfig dependencies below